default_app/preload.ts

```diff
@@ -1,4 +1,4 @@
-import { ipcRenderer, contextBridge } from 'electron/renderer';
```

Comment:
Is there anything we'd need to document in Breaking Changes about the switch from import -> const in preload here? 

Reading this, it looks like this will only affect 1) apps using esbuild/esnext that 2) also have sandboxed preload scripts, just want to make sure my read there is accurate.

Reply:
This change isn't actually anything that will impact apps, i.e. there are no breaking changes in this PR.  Every app that currently works should continue to work the exact same way.

This change is required because `default_app/package.json` now has `type: module` which means that tsc no longer replaces import with require when it builds the files.  Sandboxed preloads don't support import though so we have to the replacement instead.